pkg/service/backup/service_backup_integration_test.go

```diff
@@ -2120,141 +2119,6 @@ func TestValidateIntegration(t *testing.T) {
 	}
 }
 
-func TestBackupRestoreIntegration(t *testing.T) {
```

Comment:
@Michal-Leszczynski why this test is removed ?

Reply:
It's described in the commit message.

Reply:
got it, it's ansible script